### PR TITLE
Nbo/redirect when clicking scenario name sdcosmo 1274

### PR DESCRIPTION
**Pull request by nborde-CSM**
Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: cypress/commons/actions/generic/ScenarioManager.js [hunk 1]
```diff
@@ -67,10 +67,29 @@ function getScenarioEditableLabelInEditMode(scenarioId, timeout = 5) {
     timeout: timeout * 1000,
   });
 }
+
+function getScenarioEditButton(scenarioId) {
+  return getScenarioAccordion(scenarioId).find(GENERIC_SELECTORS.scenario.manager.button.edit);
```
Comment (csm-thu) [inline cypress/commons/actions/generic/ScenarioManager.js:72]: Could we rename `getScenarioEditButton` to `getRenameScenarioButton`?
And `GENERIC_SELECTORS.scenario.manager.button.edit` to `GENERIC_SELECTORS.scenario.manager.button.renameScenario` ?

File: cypress/e2e/brewery/RenameScenario.cy.js [hunk 2]
```diff
@@ -48,13 +48,8 @@ describe('Create scenario and rename it', () => {
     Scenarios.createScenario(scenarioName, true, SCENARIO_DATASET, SCENARIO_RUN_TEMPLATE).then((data) => {
       const scenarioId = data.scenarioCreatedId;
       ScenarioManager.switchToScenarioManager();
-      ScenarioManager.getScenarioEditableLabel(scenarioId)
-        .click()
-        .type('{selectAll}{backspace}' + newScenarioName + '{esc}') // Do not confirm new name
-        .should('have.text', scenarioName);
-
```
Comment (csm-thu) [inline cypress/e2e/brewery/RenameScenario.cy.js:55]: You've removed the part testing that we can cancel the operation of renaming a scenario, was it on purpose? If we can still cancel the renaming of a scenario by pressing "Escape", can't we keep this part of the test?

Comment (nborde-CSM) [reply cypress/e2e/brewery/RenameScenario.cy.js:55]: It's a mess. I removed this part because the test didn't pass when at the first trials.

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Pull request merged**